### ISSUE TITLE
How can we encourage everyone to keep contributing on github/publishing thoughts?

### ISSUE THREAD

**Issue by devinhalladay** [GD399-OSD/GD399-Meta#6]
I feel like I'm seeing a little activity from a few people in the class but from the majority I haven't seen or heard anything…maybe it would be good to have a convo this friday about enforcing participation? Even if it's just making an empty repo and spitting stream-of-consciousness into it, that's better than nothing. 

Thoughts?

**Comment by bjornmeansbear** [comment]
Yes, valuable point. Chris and John popup on their font repo, but otherwise its you and I

Now, that does mirror how most communities work — one or two or a small select group do most things; but right now there is 0 from others … Hopefully getting a project started this week will also help with that.

**Comment by devinhalladay** [comment]
Yeah that's a pretty normal thing for open source communities…maybe we can try to change that, at least for the purposes of this course.

Plus I think once everyone is involved and doing their own thing, you won't need to be quite as involved outside of class time because I know you've got a ton of stuff going on outside this course.

**Comment by bjornmeansbear** [comment]
It would be wonderful if we can all solve each others problems via the issues — the even minor discussions we've had over small org things have been fun and helpful. So hopefully this is a good way for us all to interact when not in class together. Or even when we are in class together!

**Comment by devinhalladay** [comment]
For sure! Also, great idea assigning people to #7 to encourage participation. That could be a good way to make everyone feel accountable!

Let's keep this issue open for now, and if we manage to talk more about this on Friday we can add our conclusions and close it then.
